### ISSUE TITLE
Allow filtering of as_of queryset

### ISSUE THREAD

**Issue by rossmechanic** [jazzband/django-simple-history#397]
Closing #229 and accepting this story to be completed in a separate PR. Will use the implementation from https://github.com/treyhunner/django-simple-history/compare/manager-to-queryset and add tests and then put up PR.

**Comment by williamcaruso** [comment]
Great idea! When is this feature expected to be released?

**Comment by rossmechanic** [comment]
Hey @williamcaruso ! Sorry for the delayed response. Let me take a look at this tomorrow and give you a better sense of what we want here.

**Comment by rossmechanic** [comment]
@williamcaruso just looked at this. Not really sure what you're asking about those two options. Can you clarify? Going to try to get a PR for this fix up in the next few weeks, but you're welcome to take it on with https://github.com/treyhunner/django-simple-history/compare/manager-to-queryset as a starting point.

**Comment by niksite** [comment]
Any updates on this PR? I am currently want to do some aggregation/annotation on historical records as of some point in time and queryset instead of generator would be very of help here.

**Comment by jordonwii** [comment]
This issue is blocking a feature I'm working on. What would it take to get https://github.com/jazzband/django-simple-history/pull/229 checked in to unblock at least a simple version of this feature?

I may be able to make the changes if you can help me identify what would need to be updated in that PR (is Django 1.6 support still an issue?).

**Comment by rossmechanic** [comment]
Hey @jordonwii, We don't support 1.6. Earliest version we support is 1.11 (and we'll be removing support for that as soon as I can get around to it). If you want to take Trey's manager -> queryset work and put up an updated PR, I'd be happy to look at it.

**Comment by dwasyl** [comment]
@rossmechanic Would the functionality proposed here work on related fields (as mentioned in #715?). If so, I'll just close that issue and let this one carry-on. But either way, huge +1 to this feature as well.
